CMakeLists.txt

```diff
@@ -152,7 +154,8 @@ else()
 install(TARGETS ${PROJECT_NAME}
     RUNTIME DESTINATION bin
 )
-install(FILES systemd/airsaned.service
+configure_file(systemd/airsaned.service.in ${CMAKE_BINARY_DIR}/airsaned.service)
```

Comment:
@JBBgameich This causes all of the environment variables on the `Exec=` line to be removed.

The `@ONLY` argument to [configure_file](https://cmake.org/cmake/help/latest/command/configure_file.html) can be used here to avoid this.

Reply:
Thanks for noticing, this should fix it: https://github.com/SimulPiscator/AirSane/pull/88/